contrib/tracing/log_utxo_coins.py

```diff
@@ -15,6 +15,7 @@
 struct coin_data
 {
   u32 height;
+  u32 index;
   char hash[MAX_HASH_SIZE];
   long value;
   u32 isCoinBase;
```

Comment:
should be `bool` too

Reply:
Did you mean to `isCoinBase`?

Cause, `index` refers to the `COutPoint::n` here https://github.com/0xB10C/bitcoin/blob/7be143a960e2bb9ac81144f55c45731c1bb209c5/src/primitives/transaction.h#L26-L30

For the `isCoinBase` which refers to `Coin::fCoinBase` it is of `unsigned int` type.
https://github.com/0xB10C/bitcoin/blob/7be143a960e2bb9ac81144f55c45731c1bb209c5/src/coins.h#L30-L37

Reply:
yes, sorry if that was unclear. For me the comment is on the `isCoinbase` line.

![image](https://user-images.githubusercontent.com/19157360/131505950-c04f70b9-bfce-445e-a143-c1b7150fdc81.png)

Reply:
You could use Coin::IsCoinBase() which returns a bool

Reply:
See https://github.com/0xB10C/bitcoin/pull/1/files?file-filters%5B%5D=.cpp#r700080149